### ISSUE TITLE
Networkx 2.0 compatibility 

### ISSUE THREAD

**Issue by Erotemic** [gtaylor/python-colormath#73]
In the add_type_conversion method of GraphConversionManager on line 124 of color_conversions.py there is a call to networkx `add_edge` that does not work in the new version 2.0. This can be fixed by modifying the function as follows: 

```python
    def add_type_conversion(self, start_type, target_type, conversion_function):
        super(GraphConversionManager, self).add_type_conversion(start_type, target_type, conversion_function)
        if networkx.__version__.startswith('2'):
            self.conversion_graph.add_edge(start_type, target_type, conversion_function=conversion_function)
        else:
            self.conversion_graph.add_edge(start_type, target_type, {'conversion_function': conversion_function})

```

**Comment by gtaylor** [comment]
I'd accept a PR for this

**Comment by gtaylor** [comment]
Fixed in #75.
